processing/src/main/java/io/druid/query/filter/SelectorDimFilter.java

```diff
@@ -137,6 +139,34 @@ public boolean applyLong(long input)
             };
           }
         }
```

Comment:
In the method which ends with this brace `Predicates.equalTo()` and singleton could be used.